pyramid_okta/utils.py

```diff
@@ -186,61 +202,96 @@ def clear_session(session_id):
     else:
         return True
 
+def get_groups(filter='', limit=None, app_id=None):
```

Comment:
are these groups permissions now ? does it mean no more hierarchical permissions ?